subprojects/docs/src/docs/release/notes.md

```diff
@@ -143,6 +165,12 @@ Putting processors on the compile classpath or using an explicit `-processorpath
 
 The `java-base` plugin will now add an `<sourceSetName>AnnotationProcessor` configuration for each source set. This might break when the user already defined such a configuration. We recommend removing your own and using the configuration provided by `java-base`. 
 
+### No input property hashes on the info log
+
+Gradle does not print out input property hashes when the `--info` log is enabled.
+Only the build cache key is reported on `--info`.
+Logging input property hashes is now enabled by the Gradle property `org.gradle.caching.debug`.
+
```

Comment:
I think it's enough to mention this once, no?

Reply:
I removed the "possible breaking changes" part.